Bhagyashri/Correction/css/style.css

```diff
@@ -0,0 +1,109 @@
+
+body{
+background-color:#006966;
+padding-left:100px;
+padding-right:100px;
+margin:0px;
+}
+
+.first-sect1{
+	   border-top:0px;
+	   padding:10px;
+       background-color:#ffffff;
+	   min-height:40px;
+           }
+		   
+.class-hg1{
+	float:left;
+	display:inline-block;
+	  }
+	  .class-hg1 img{  
+	  width:150px;
+	  }
+.class-hg2{
+	text-transform:uppercase;
+	float:right;
+	display:inline-block;
+	}
+nav{
+	color:#008080;
+	padding:10px;
+}	
+
+a{
+	text-decoration:none;
+	padding-left:10px;
+	padding-right:10px;
+ }
+ .class-h3{
+	padding-top:30px;
+	padding-bottom:30px;
+ }
+
+.class-h31{
+
+	margin-bottom:4px;
+	text-align:center;
+	font-size=20px;
```

Comment:
font-size: 20px; 
you had use equal to operator here.

Reply:
changed